src/test/java/ReadingRangeCountTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+
+public class ReadingRangeCountTest {
+    ReadingRangeCount readingRangeCount = new ReadingRangeCount();
```

Comment:
the need is to find continuous ranges within a given input and then print the number of elements within each range.
But since you have implemented in a different way, the comment for this implementation is as follows,
1. input data should not be hardocded.
2. input data should contain a variations with only one element also